### DISCUSSION TITLE
constant logout every few days

### DISCUSSION THREAD

**Discussion by scottydulton** [CastagnaIT/plugin.video.netflix#1093] [General]
is it normal that i have to use the authentication key method every few days or am i doing something wrong? ive installed the addon on both PC and Firestick on both 18.9 and matrix and still get logged out.

**Comment by CastagnaIT** [comment]
It is not normal,
but the current login system is not always perfect on add-on side due to netflix website peculiarities, then can left some problems like this, and it is not guaranteed that it can be solved.

if you want collaborate, you can provide log info to try analyze the situation,
then you have to follow instructions exactly:

Only on Windows machine with Kodi 19 Matrix:
- Extract the xml file from [advancedsettings.zip](https://github.com/CastagnaIT/plugin.video.netflix/files/6043272/advancedsettings.zip)
 and place it on Kodi folder: `C:\Users\[your user name]\AppData\Roaming\Kodi\userdata\`, this allow you to have Kodi debug enabled all times without on screen info.
- Open Kodi and if needed Login again on netflix with auth key
- Open add-on Expert settings, and turn on `enable debug logging`

Now you have the Kodi/add-on ready to store the log info.

Close Kodi again and reopen it, open netflix add-on, then close Kodi, now attach on message here the file on path: `C:\Users\[your user name]\AppData\Roaming\Kodi\kodi.log`

Now you will have to wait next time that the problem happen again, and the first time it happens, close Kodi and attach here the new "kodi.log" file.

NOTE: the kodi.log file will be cleared every time you close/open Kodi

**Comment by jakermx** [comment]
How many devices plan do you have?

I have the 4 dev plan, and I never have got that  issue...I could be due FIFO #  nline devices or cookie/setting based ogout timeout

Please share your log
